share/locale/languages.xml

```diff
@@ -37,6 +37,7 @@
   <language code="it" name="Italiano" />
   <language code="ja" name="日本語" />
   <language code="ka" name="ქართული" />
+  <language code="kab" name="ⵜⴰⵇⴱⴰⵢⵍⵉⵜ" />

```

Comment:
Let's go for just "Taqbaylit", see also https://de.wikipedia.org/wiki/Kabylische_Sprache